### ISSUE TITLE
defining macros

### ISSUE THREAD

**Issue by eri451** [tmux/tmux#983]
tl;dr: I would be nice to be able to define some sort of macros.

I like to keep a session with interactive programs as background session ("apps"). And even get some into specific panes of an attached session.
It's super nice that tmux does that already like so:
     $ tmux swap-pane -d -s apps:2.0

but since I would like to grab multiple applications that way, wondering which window holds what app again it is not very nice. But macros would easily solve this.
So someone could go:
     $ tmux macro app1 swap-pane -d -s apps:1:0
    $ tmux macro app2 swap-pane -d -s apps:5:0
and so
    $ tmux app1
would apply the command.

I'm running:
$ uname -srv && tmux -V && echo $TERM
Linux 4.9.0-3-amd64 #1 SMP Debian 4.9.30-1 (2017-06-04)
tmux 2.3
xterm-256color

**Comment by nicm** [comment]
You can write a script and then set it as a command alias.

**Comment by eri451** [comment]
then I have to call 
    :so myscript<cr>
this would just work if the script is in my current working directory
